### DISCUSSION TITLE
Animate consistent vector stroke width

### DISCUSSION THREAD

**Discussion by Stickyricesticky** [keiwando/toonsquid#604] [General]
Any advice to animate vector strokes that hold a consistent line width? Eg a circle animating from small to big & keeping the stroke width consistently at 1pt? I’ve tried hacking it a bit with adjusting stroke width for first and last keyframe but not working too well. Thanks

**Comment by budaloops** [comment]
That’s actually the way to do it. Set the stroke width to 1 on the first keyframe, and set it to 1 on the last keyframe and it will be consistent. What problems did you get when you did it this way?
